crazyflie/scripts/gui.py

```diff
@@ -0,0 +1,222 @@
+
+            # no update for a while -> mark red
+            if time.time() - self.watchdog[name] > 2.0:
+                robotmodel.material('#ff0000')
+
+    def on_vis_click(self, e: events.SceneClickEventArguments):
+        hit = e.hits[0]
+        name = hit.object_name or hit.object_id
+        ui.notify(f'You clicked on the {name}')
+        if name == 'ground':
+            self.tabpanels.value = 'all'
+        else:
+            self.tabpanels.value = name
+
+    def on_status(self, msg, name) -> None:
+        status_ok = True
+        supervisor_text = ""
+        if msg.supervisor_info & Status.SUPERVISOR_INFO_CAN_BE_ARMED:
+            supervisor_text += "can be armed; "
+        if msg.supervisor_info & Status.SUPERVISOR_INFO_IS_ARMED:
+            supervisor_text += "is armed; "
+        if msg.supervisor_info & Status.SUPERVISOR_INFO_AUTO_ARM:
+            supervisor_text += "auto-arm; "
+        if msg.supervisor_info & Status.SUPERVISOR_INFO_CAN_FLY:
+            supervisor_text += "can fly; "
+        if msg.supervisor_info & Status.SUPERVISOR_INFO_IS_FLYING:
+            supervisor_text += "is flying; "
+        if msg.supervisor_info & Status.SUPERVISOR_INFO_IS_TUMBLED:
+            supervisor_text += "is tumpled; "
+            status_ok = False
+        if msg.supervisor_info & Status.SUPERVISOR_INFO_IS_LOCKED:
+            supervisor_text += "is locked; "
+            status_ok = False
+        self.supervisor_labels[name].set_text(supervisor_text)
+
+        battery_text = f'{msg.battery_voltage:.2f} V'
+        if msg.battery_voltage < 3.8:
+            status_ok = False
+        if msg.pm_state == Status.PM_STATE_BATTERY:
+            battery_text += " (on battery)"
+        elif msg.pm_state == Status.PM_STATE_CHARGING:
+            battery_text += " (charging)"
+        elif msg.pm_state == Status.PM_STATE_CHARGED:
+            battery_text += " (charged)"
+        elif msg.pm_state == Status.PM_STATE_LOW_POWER:
+            battery_text += " (low power)"
+            status_ok = False
+        elif msg.pm_state == Status.PM_STATE_SHUTDOWN:
+            battery_text += " (shutdown)"
+            status_ok = False
+        self.battery_labels[name].set_text(battery_text)
+
+        radio_text = f'{msg.rssi} dBm; Unicast: {msg.num_rx_unicast} / {msg.num_tx_unicast}; Broadcast: {msg.num_rx_broadcast} / {msg.num_tx_broadcast}'
+        self.radio_labels[name].set_text(radio_text)
+
+        if status_ok:
+            self.robotmodels[name].material('#00ff00')
+        else:
+            self.robotmodels[name].material('#ff0000')
+
+        self.watchdog[name] = time.time()
+
+
+    def on_tab_change(self, arg):
+        for name, robotmodel in self.robotmodels.items():
+            if name != arg.value:
+                robotmodel.scale(1)
+        if arg.value in self.robotmodels:
+            self.robotmodels[arg.value].scale(2)
+
+    def on_reboot(self, name) -> None:
```

Comment:
I guess we would like to have even more buttons in the future, but for clarity lets leave out buttons that haven't been implemented perhaps?